BeagleBone/AI/pru/blinkR30.pru1_1.c

```diff
@@ -1,11 +1,11 @@
 ////////////////////////////////////////
 //	blinkR30.c
-//	Blinks LEDs wired to P9_25 (and others) by writing register R30 on the PRU
+//	Blinks LEDs wired to pr1_pru1 pins by writing register R30 on the PRU
 //	Wiring:	P9_16 connects to the plus lead of an LED.  The negative lead of the
 //			LED goes to a 220 Ohm resistor.  The other lead of the resistor goes
 //			to ground.
 //	Setup:	None
-//	See:	prugpio.h to see which pins attach to R30
+//	See:	cloud9/common/prugpio.h for which pins attach to pr1_pru1.
```

Comment:
I think the "cloud9" in the path is a bit confusing.

Reply:
It was really helpful for me so i added it, should i remove it in next PR?